ui/tests/integration/components/replication-dashboard-test.js

```diff
@@ -0,0 +1,56 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+const REPLICATION_DETAILS = {
+  state: 'stream-wals',
+  primaryClusterAddr: 'https://127.0.0.1:8201',
+};
+
+const REPLICATION_DETAILS_SYNCING = {
+  state: 'merkle-diff',
+  primaryClusterAddr: 'https://127.0.0.1:8201',
+};
+
+module('Integration | Enterprise | Component | replication-dashboard', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.set('replicationDetails', REPLICATION_DETAILS);
+    this.set('replicationDetailsSyncing', REPLICATION_DETAILS_SYNCING);
+    this.set('componentToRender', 'replication-secondary-card');
```

Comment:
same as my previous nit: variables that are only used once should be set inside the individual tests inside of the hooks